FireRender.Maya.Src/FireRenderVolume.cpp

```diff
@@ -1735,8 +1755,7 @@ bool NorthstarFluidVolume::TranslateVolume(void)
 	
 	// apply nodes to main shader
 	volumeShader.xSetValue(RPR_MATERIAL_INPUT_DENSITYGRID, densityGridNode);
-	//volumeShader.xSetValue(RPR_MATERIAL_INPUT_DENSITY, densityLookupNode); // not implemented in northstar
-	volumeShader.xSetParameterF(RPR_MATERIAL_INPUT_DENSITY, 10000.f, 1.f, 1.f, 1.f); // constant mock
+	volumeShader.xSetParameterF(RPR_MATERIAL_INPUT_DENSITY, 1000.f, 1.f, 1.f, 1.f); // multiplication coefficient
```

Comment:
Why do we still need a 1000 constant multiplier if we modify grid values directly?

If yes, it was 10 000 in previous version. Was this change intentional?